api/v1/tigerastatus_types.go

```diff
@@ -104,6 +104,27 @@ type TigeraStatusList struct {
 	Items           []TigeraStatus `json:"items"`
 }
 
+// TigeraStatusReason represents the reason for a particular condition. A reason may be one of the below.
+type TigeraStatusReason string
+
+const (
+	AllObjectsAvailable       TigeraStatusReason = "AllObjectsAvailable"
```

Comment:
nit: Only the first line needs to declare the type. If the first is done, the rest will automatically be assumed:

Example:
```
	ResourceNotReady  TigeraStatusReason = "ResourceNotReady"
	PodFailure = "PodFailure"
```